doc/1/plugins/guides/events/core-hotelclerk-addSubscription/index.md

```diff
@@ -0,0 +1,29 @@
+---
+code: true
+type: page
+title: core:hotelClerk:addSubscription
+---
+
+# core:hotelClerk:addSubscription
+
+
+| Arguments  | Type              | Description                           |
+| ---------- | ----------------- | ------------------------------------- |
+| `diff`     | <pre>object</pre> | Contains information about subscription |
+
+Triggered whenever a [subscription](/core/1/api/controllers/realtime/subscribe) is added.
+
+## diff
+
+The provided `diff` object has the following properties:
+
+| Properties     | Type                 | Description                                                                                                       |
+| -------------- | -------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `roomId`       | <pre>integer</pre>   | New room unique identifier                                                                                    |
+| `connectionId` | <pre>integer</pre>   | [ClientConnection](/core/1/protocols/api/context/clientconnection) unique identifier                          |
+| `index`        | <pre>string</pre>    | Index                                                                                                             |
+| `collection`   | <pre>string</pre>    | Collection                                                                                                        |
+| `filters`      | <pre>object</pre>    | [Filters](https://docs.kuzzle.io/core/1/guides/cookbooks/realtime-api/terms/) to match with                       |
```

Comment:
As discussed together yesterday, this is not exactly the filters provided by the subscribing users. 

```suggestion
| `filters`      | <pre>object</pre>    | Filters in [Koncorde's normalized format](https://www.npmjs.com/package/koncorde#filter-unique-identifier)  |
```